compiler-rt/lib/radsan/radsan_context.cpp

```diff
@@ -46,10 +44,10 @@ void Context::expectNotRealtime(const char *intercepted_function_name) {
   if (inRealtimeContext() && !isBypassed()) {
     bypassPush();
     printDiagnostics(intercepted_function_name);
-    if (get_error_action_() == OnErrorAction::ExitWithFailure) {
-      exit(EXIT_FAILURE);
-    }
-    bypassPop();
+
+    // TODO: re-implement "continue" with normal sanitizer options
+    exit(EXIT_FAILURE);
```

Comment:
We now just die without any option to do anything otherwise. This is a loss of functionality in favor of getting C back up and running. I also think this is the most reasonable default of the modes we have right now